### ISSUE TITLE
Geometric tinkering - many images lost resolution 

### ISSUE THREAD

**Issue by cukierm** [parkmath/parkmath#44]
Before I take a bunch of screenshots, save them, and upload them... is there a more automated fix?

**Comment by anandthakker** [comment]
We can try to recover the original images from the InDesign folder for this lesson, similar to the shape & size thing I did recently.  @cukierm if you can stick the folder in dropbox and let me know when it's there, I should be able to convert them quickly/easily.

**Comment by cukierm** [comment]
Just put the 200 MB “geometry” folder in dropbox - don’t think it’s gonna transfer with the internet in this coffeeshop. Home in an hour, probably.
OK, it is transferring slowly, but I’ve still gotta leave soon!

**Comment by cukierm** [comment]
Yep... half the files are transferred, but you probably need all of them to
be there to do anything. I'll email in an hour or two when I have internet
access again.

**Comment by cukierm** [comment]
All there now!
